python/cuml/metrics/cluster/__init__.py

```diff
@@ -24,5 +24,5 @@
 from cuml.metrics.cluster.entropy import cython_entropy as entropy
 from cuml.metrics.cluster.silhouette_score import \
     cython_silhouette_score as silhouette_score
-from cuml.metrics.cluster.silhouette_score import \
-    cython_silhouette_samples as silhouette_samples
+# from cuml.metrics.cluster.silhouette_score import \
+#     cython_silhouette_samples as silhouette_samples
```

Comment:
It would be good to reference the issue here, should we still keep the cython code compiling/in the package? Not a big deal, but wanted to ask

Reply:
Yep! I'll add a comment.

In terms of keeping the cython code, this was my compromise between total removal and just adding a warning. If somebody *really* wants to go digging and *really* needs silhouette_score for a small number of samples, it's still there for them. It also means less code churn, since the cython bindings will not be changed once the C++ implementation is fixed.